kivy/uix/textinput.py

```diff
@@ -176,10 +176,12 @@ def insert_text(self, substring, from_undo=False):
 
 
 if 'KIVY_DOC' in environ:
-    def triggered(*_):
+    def triggered(*_, **__):
         def func(*args, **kwargs):
             return func(*args, **kwargs)
         return func
```

Comment:
I think the problem is that it is named `func` but both return `triggered`.

Reply:
yeah, i was tired i guess, max recursion 😆